frontend/src/helpers/dataOverlay.js

```diff
@@ -3,17 +3,20 @@ const DEFAULT_METABOLITE_COLOR = '#9df';
 const DEFAULT_REACTION_COLOR = '#fff';
 
 const DATA_TYPES_COMPONENTS = {
-  transcriptomics: {
+  gene: {
+    description: 'gene (e.g., transcriptomics, proteomics, etc.)',
     componentType: 'gene',
     className: 'enz',
     defaultColor: DEFAULT_GENE_COLOR,
   },
-  metabolomics: {
+  metabolite: {
+    description: 'metabolite (e.g., metabolomics, etc.)',
     componentType: 'metabolite',
     className: 'met',
     defaultColor: DEFAULT_METABOLITE_COLOR,
   },
-  fluxomics: {
+  reaction: {
+    description: 'reaction (e.g., reaction presence, fluxomics, kcat, etc.)',
```

Comment:
I'm not sure if it is the best way to follow the data type with examples. 
Maybe we can show the data type as just `gene` or `metabolite` while add a `info` icon to provide details.

Reply:
NIce, this gave me an idea for a new commit, so I'm marking the PR as draft until then.

Reply:
Based on your and @inghylt's feedback, I have reimplented this by moving the data type description below the dropdown in commit [4a44ab2](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1181/commits/4a44ab256970a9a37ce7235905d3c3c5139d0c8f):

<img width="334" alt="image" src="https://user-images.githubusercontent.com/23480589/199070605-d6859c8e-6b00-47b5-8b94-a67fa7e05d72.png">

Based on this, I'll go ahead and request another review.

Reply:
This was resolved across several commits, and finally approved by @nanjiangshu.